### ISSUE TITLE
Must refine calls to least_busy better

### ISSUE THREAD

**Issue by nonhermitian** [Qiskit/qiskit-textbook#110]
Calls like this one in the grover chapter:
```python
provider = IBMQ.load_account()
device = least_busy(provider.backends(simulator=False))
```
now can bring up the 1Q Armonk device that fails to run the algorithm.  Something like `filters=lambda b: b.configuration().n_qubits >= 5` should also be added

**Comment by frankharkins** [comment]
Fixed in PR#176, will keep an eye out for more problems.

**Comment by HuangJunye** [comment]
This problem is fixed in 3-qubit example but not yet in 2-qubit example: https://qiskit.org/textbook/ch-algorithms/grover.html#2.1.2-Experiment-with-Real-Devices-
